### DISCUSSION TITLE
Fuck Russia

### DISCUSSION THREAD

**Discussion by ElementShadow** [Tyrrrz/YoutubeDownloader#299] [Q&A]
But at same time fuck you brother, why using your program should be so complicated? How even "set enviroment variable" wtf is this.
No guide, no tutorial

**Comment by Tyrrrz** [comment]
The easiest way is:

1. Press `Win+R`
2. Type in `sysdm.cpl` and press `Enter`
3. Open the `Advanced` tab
4. Click `Environment Variables`

Steps 1-2 are a shorthand for opening advanced system settings, which can be different between Windows 10 and 11.

**Comment by remiX-** [comment]
What is this nonsense, this downloader is the easiest thing to use. Thanks Tyrrrz for you amazing work :D

**Reply by Tyrrrz** [reply]
They were talking about setting an environment variable, which is necessary if you're trying to use the software from russia

**Reply by ElementShadow** [reply]
Funny part about it, am from Ukraine, and still its ask for "environment variable", i assume because i have russian letters

**Reply by Tyrrrz** [reply]
It's most likely because you're using russian as the system language, which makes Windows assume that's your locale. You can try using the latest CI build from [here](https://github.com/Tyrrrz/YoutubeDownloader/actions/runs/3469683511) where I changed the checks a bit to accomodate that.

**Comment by toxaold** [comment]
Да, классная идея. Жалко что исправляется заменой "ru" на другие буквы в конфиге.
